com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/DoFGather.compute

```diff
@@ -270,7 +296,7 @@ void KMain(uint3 dispatchThreadId : SV_DispatchThreadID)
     // back to front alpha blending of the near, far and in-focus buckets
     for (int j = 0; j < NUM_BUCKETS; ++j)
     {
-        float alpha = saturate(totalColor[j].w * rcp(GetNumSamples(maxCoC[j], maxRadius)) * rcp(GetSampleWeight(maxCoC[j])));
+        float alpha = saturate(totalColor[j].w * 2 * rcp(GetNumSamples(maxCoC[j], maxRadius))* rcp(GetSampleWeight(maxCoC[j])));
```

Comment:
Why the * 2 here?

Reply:
I have added the explanation in the code, since any reply here will not be visible for long.

Reply:
👍